yaksh/templates/yaksh/question.html

```diff
@@ -155,25 +152,31 @@
                  <div class="panel-body">
                      {% if question.type == "mcq" %}
                      {% for test_case in test_cases %}
-                     <input name="answer" type="radio" value="{{ test_case.options }}" />{{ test_case.options|safe }} <br/>
+                     {% if last_attempt and last_attempt|safe == test_case.options|safe %}
```

Comment:
last attempted answer for MCC questions is not checked.

Reply:
MCC is checked now